src/runtime/virtcontainers/clh.go

```diff
@@ -711,7 +714,11 @@ func (clh *cloudHypervisor) StartVM(ctx context.Context, timeout int) error {
 	}
 	clh.state.PID = pid
 
-	ctx, cancel := context.WithTimeout(ctx, clh.getClhAPITimeout()*time.Second)
+	bootTimeout := clh.getClhAPITimeout()
+	if bootTimeout < clhCreateAndBootVMMinimumTimeout {
+		bootTimeout = clhCreateAndBootVMMinimumTimeout
+	}
+	ctx, cancel := context.WithTimeout(ctx, bootTimeout*time.Second)
```

Comment:
@danmihai1, conceptually this is correct.  I'm wondering only on whether we could simply use the clhTimeout instead though.  Let me know your thoughts on this and why you decided to add a specific new var.

Reply:
@fidencio, my preference as a Kata relative newcomer would be to replace all of the timeouts below with a single number between 10 and 20. I didn't propose that just because I didn't want to rock the boat more than I had to - given that I am not familiar with a lot of the Kata history. Please let me know in case you think simplifying the code by reducing the number of Timeout variables would acceptable.

clhTimeout is a harder to understand name compared with clhCreateAndBootVMMinimumTimeout. Also, clhTimeout seems to be intended strictly for the CLH API to become available - not intended as an API call timeout. But again, I don't know the history here, so reusing clhTimeout would be OK with me too.

	clhTimeout                     = 10
	clhAPITimeout                  = 1
	clhAPITimeoutConfidentialGuest = 20
	// Timeout for hot-plug - hotplug devices can take more time, than usual API calls
	// Use longer time timeout for it.
	clhHotPlugAPITimeout                   = 5
	clhStopSandboxTimeout                  = 3
	clhStopSandboxTimeoutConfidentialGuest = 10

Thanks!